src/figuresClasses.ts

```diff
@@ -1,11 +1,86 @@
-export interface Figure {}
+export interface Figure {
+  shape: 'triangle' | 'circle' | 'rectangle';
+  color: 'red' | 'green' | 'blue';
+}
+
+function isTriangle(a: number, b: number, c: number): boolean {
+  if (a + b <= c || a + c <= b || b + c <= a) {
+    return true;
+  }
+
+  return false;
+}
+
+export class Triangle implements Figure {
+  constructor(
+    public color: Figure,
```

Comment:
[CODE STYLE] - The `color` property should be of type `string` representing the color value ('red', 'green', 'blue'), not `Figure`.